generators/entity/templates/server/src/main/java/package/web/rest/_EntityResource.java

```diff
@@ -130,7 +147,7 @@
      * @return the ResponseEntity with status 200 (OK) and the list of <%= entityInstancePlural %> in body
      */
     @GetMapping("/<%= entityApiUrl %>")
-    @Timed<%- include('../../common/get_all_template', {viaService: viaService}); -%>
+    @Timed<%- include('../../common/get_all_template', {viaService: viaService}) -%>
```

Comment:
We should probably also have an endpoint producing `application/stream+json` to prevent blocking in large JSON serialization/deserialization